lib/plotutil.py

```diff
@@ -58,3 +58,40 @@ def plot(func, xstart, xend, step, color_name, xlabel, ylabel, gui):
 		plt.savefig(".temp/generated_plot.png")
 	plt.cla()
 	plt.clf()
+
+def plot_line(arrays, color_name, xlabel, ylabel, gui):
+	# show plot summary
+
+	print '***** Plot Summary *****'
+	print 'Arrays: {}'.format(arrays)
+	print 'Color: {}'.format(color_name)
+	print 'X-label: {}'.format(xlabel)
+	print 'Y-label: {}'.format(ylabel)
+
+	try:
+		#check if color is hex code
+		is_hex = re.search(r'^#(?:[0-9a-fA-F]{3}){1,2}$', color_name)
+		if not is_hex:
+			colors = mcolors.cnames
+			if not color_name in colors:
+				print color_name,": Color not found."
+				color_name = 'blue'
+		xvals = list(map(float, arrays[1:arrays.find(']')].split(','))) #extract numbers from X-array
+		yvals = list(map(float,arrays[arrays.find(']')+3:len(arrays)-1].split(','))) #extract numbers from Y-array
+		if len(xvals) == len(yvals):
+			plt.plot(xvals, yvals, color=color_name, linewidth=2.0)
+		else:
+			print('Error: You need same number of X and Y values')
+		plt.xlabel(xlabel)
+		plt.ylabel(ylabel)
```

Comment:
Put this within the len check if block

Reply:
@Aniq55 ok changed it. is it done with the commit or need i do something with the pr here that the commit is linked to this pr? some git related stuff is still a bit confusing for me :D